packages/server/graphql/private/mutations/toggleAllowInsights.ts

```diff
@@ -0,0 +1,41 @@
+import {r} from 'rethinkdb-ts'
+import {getUsersByEmails} from '../../../postgres/queries/getUsersByEmails'
+import {MutationResolvers} from '../resolverTypes'
+
+const toggleAllowInsights: MutationResolvers['toggleAllowInsights'] = async (
```

Comment:
+1 we could use the [isCompanyDomain](https://github.com/ParabolInc/parabol/blob/4d979ad66a68ad399864f1fcc393cff4bd642b5e/packages/server/utils/isCompanyDomain.ts) function you created to make sure that no one adds `gmail` as a domain. It's a private mutation, so it's very unlikely that one of us would do that, but it would mean that a user could see data for all gmail users